scripts/generate_driver_wrappers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""This script is required for the auto generation of the
+   psa_crypto_driver_wrappers.c file"""
+
+import sys
+import os
+import jinja2
+
+def render(template_path: str) -> str:
+    environment = jinja2.Environment(
+        loader=jinja2.FileSystemLoader(os.path.dirname(template_path)),
+        keep_trailing_newline=True)
+    template = environment.get_template(os.path.basename(template_path))
+    return template.render()
+
+N = len(sys.argv)
+if N != 2:
+# This is the Root directory.
+    ROOT_DIR = ""
```

Comment:
Considering that a script can be invoked as `python3 some_dir/scipt.py` or `cd some_dir && python3 script.py` it may be more portable to anchor it using the __file__ attribute since we already know where that script is relative to the project

~~~
# Python3 (not backwards compatbile)
import pathlib
pathlib.Path(__file__).parent.resolve()

# Python2 and 3
import os
os.path.dirname(os.path.abspath(__file__))
~~~